omero/sysadmins/cli/data-ownership.txt

```diff
@@ -0,0 +1,53 @@
+Changing the ownership of data
```

Comment:
My suggestion was to delete this page entirely and simply link directly from the "CLI and a client" page to the pages named "Changing the ownership of data" and "Moving data between groups"?